README.md

```diff
@@ -1,3 +1,27 @@
 # Northcoders News API
 
-If you want to re-create the project you need to create two files .env.test env.development and set an evironment variable PGDATABASE=nc_news or PGDATABASE=nc_news_test in both with the database names referred to in setup.sql
\ No newline at end of file
+### Hosted Version
+https://nc-news-a8e3.onrender.com/api
+
+### Project Outline
+This project is a news outlet where you can create delete and update a plethora of things including but not limited to comments, articles and topics.
```

Comment:
Kind of true - but I think this outline could be made a bit more effective.
We've built an API to access news data programmatically 
It's not really a news outlet in itself yet! (stay tuned for the frontend, however 😄)

Might be worth a little rewrite here